Softeq.NetKit.Services.PushNotifications/Helpers/DeviceRegistrationConverter.cs

```diff
@@ -17,12 +17,32 @@ public static DeviceRegistration Convert(RegistrationDescription description)
                 return null;
             }
 
-            return new DeviceRegistration
+            var deviceRegistration = new DeviceRegistration
             {
                 Tags = new List<string>(description.Tags),
                 RegistrationId = description.RegistrationId,
                 Platform = platform
             };
+
+            switch (platform)
+            {
+                case PushPlatformEnum.iOS:
+                {
+                    deviceRegistration.PnsHandle = ((AppleTemplateRegistrationDescription) description).DeviceToken;
+                    break;
+                }
+                case PushPlatformEnum.Android:
+                {
+                    deviceRegistration.PnsHandle = ((GcmRegistrationDescription)description).GcmRegistrationId;
```

Comment:
Shouldn't this be GcmTemplateRegistrationDescripion?

Reply:
fixed